### ISSUE TITLE
Simulate light emission remove hardcoded illuminator 1 coordinates

### ISSUE THREAD

**Issue by tobiaskleiner** [gammasim/simtools#1049]
Use the illuminator coordinates from the DB as soon as they are uploaded for the different illuminators. (for the use with the layout option).

**Comment by GernotMaier** [comment]
@tobiaskleiner - is this still an issue? I think we have added the illuminator positions to the database now. Are they still hardcoded?

**Comment by tobiaskleiner** [comment]
Yes we have added the coordinates to the model parameters in gitlab. I need to see if they are now in the DB to remove the hardcoded values.

**Comment by tobiaskleiner** [comment]
Closing this one as it is fixed with #1091.
